.github/workflows/doxygen.yml

```diff
@@ -39,6 +38,10 @@ jobs:
           if [ ! -f doxygen.err ]; then echo "No Doxygen error file found!"; exit 1; fi
           if [ -s doxygen.err ]; then echo "Doxygen failed with the following error(s):"; cat doxygen.err; exit 1; else echo "Doxygen passed."; fi
 
+  deploy:
```

Comment:
Not completely. I guess you cannot access the created data of the `build` step here. Either GitHub allows you to do that somehow or you have generate the documentation again.

Reply:
Okay, do you mind if we try #13 first?